[News] 2020-10-07 (#508)
sider/sider-docs: docs/news/2020.md

---

## Recommended ruleset for cpplint

<time class="news-date" datetime="2020-10-07">October 7, 2020</time>

> Sider will release our recommended ruleset for cpplint in mid October 2020. Prior to the release, the notification message will be shown on the Tools tab if the cpplint configuration file `CPPLINT.cfg` doesn't exist in your repository.

Reviewer: [suggestion]

- "the Tools tab" is too detailed. There are some users to use Sider only on GitHub.
- How about adding the link as follows?

```suggestion
Sider will release our recommended ruleset for cpplint in mid October 2020. Prior to the release, the warning message will be shown if the cpplint configuration file `CPPLINT.cfg` doesn't exist in your repository.

See our [cpplint document](../../tools/cplusplus/cpplint.md#recommended-ruleset)
```

---

## Recommended ruleset for cpplint

<time class="news-date" datetime="2020-10-07">October 7, 2020</time>

> Sider will release our recommended ruleset for cpplint in mid October 2020. Prior to the release, the warning message will be shown if the cpplint configuration file `CPPLINT.cfg` doesn't exist in your repository.

Reviewer: [IMO] I'd like to use `we` instead of `Sider` because the service name is changeable (e.g. SideCI -> Sider). But this is just an opinion, so you can ignore it if you have some reasons.

```suggestion
We will release our recommended ruleset for cpplint in mid October 2020. Prior to the release, the warning message will be shown if the cpplint configuration file `CPPLINT.cfg` doesn't exist in your repository.
```

Author: Fixed. I searched 2020.md and found only one usage of 'Sider'. Others are 'we' as you said. I shall follow the grammar.

> See our [cpplint document](../tools/cplusplus/cpplint.md#recommended-ruleset).

Author: > __[remark-lint]__ Link to unknown heading in `../tools/cplusplus/cpplint.md`: `recommended-ruleset` ([view](https://sider.review/gh/repos/142577083/pulls/508#issue-4735887))
>
> <details>
>
> | Rule | Severity |
> | ---- | -------- |
> | `missing-heading-in-file` | `warn` |
>
> You can [close this issue](https://sider.review/gh/repos/142577083/pulls/508#issue-4735887) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

The section "recommended-ruleset" will be available when PR #506 is published.

---

> ## Recommended ruleset for cpplint and more

Reviewer: ```suggestion
## Recommended ruleset for cpplint
```

<time class="news-date" datetime="2020-10-07">October 7, 2020</time>

Sider will release our recommended ruleset for cpplint in mid October 2020. Prior to the release, the notification message will be shown on the Tools tab if the cpplint configuration file `CPPLINT.cfg` doesn't exist in your repository.

> In addition, this release includes the following updates:
> 
> - **ESLint** The ESLint plugin for Vue.js [`eslint-plugin-vue`](https://github.com/vuejs/eslint-plugin-vue/releases/tag/v7.0.0) is added to the default package list.
> - **RuboCop** Some [cops](https://github.com/sider/runners/pull/1506/files#diff-2d8efbdb905f4de38c35365f48a62992) are added to the optional gem list.
> 

Reviewer: Please remove these trivial changes.

```suggestion
```
